frontend/src/styles/variables.module.scss

```diff
@@ -47,7 +47,7 @@ $alt-dark: #5f6368;
 $alt-grey: #bdbdbd;
 $alt-red: #d32f2f;
 
-// colors from VEGA map
+// colors for VEGA map
```

Comment:
This actually is "from" as I meant these single colors are derived from the map. But maybe we should specify "from yellowgreen"